vtk-cdat/meta.yaml.in

```diff
@@ -1,44 +1,35 @@
+{% set build_number = @BUILD_NUMBER@ %}
+
 package:
   name: vtk-cdat
-  version: 8.0.1.@VERSION@
+  version: {{ environ['GIT_DESCRIBE_TAG'][1:] ~ "." ~ environ['GIT_DESCRIBE_NUMBER']}}.@VERSION@
 
 source:
-  git_rev: 9a839a5c50131cc30320e6fc05712572096ca6fd
-  git_url: git://github.com/Kitware/VTK.git
+  git_branch: master
+  git_url: https://gitlab.kitware.com/vtk/vtk.git
 
 build:
-  number: @BUILD_NUMBER@
-  skip: True  # [win]
-  detect_binary_files_with_prefix: true
-  has_prefix_files:                           [win]
-    - Lib/site-packages/vtk/__init__.py       [win]
-  features:  # {{{ mesalib }}}
-    - mesalib  # {{{ mesalib }}}
+  number: {{ build_number }}
+  features:
+    - mesalib  # [VTK_WITH_OSMESA]
+
 
 requirements:
   build:
+    - toolchain
     - cmake
-    - python
-    - proj4 <5
-    - hdf5 1.10.*
-    - ffmpeg
-    - freetype
-    - mesalib >17  # {{{ mesalib }}}
-    - x264
```

Comment:
we want this (x264) otherwise it's bad videos.

Reply:
Sounds good. I can add that - I wasn't sure what was that for.